HALL_OF_FAME.md

```diff
@@ -1,3 +1,5 @@
 # Hall of fame
 
 Grafana Labs would like to thank the following security researchers for having reported those vulnerabilities:
+
+[renniepak](https://github.com/renniepak) - 💰💰
```

Comment:
Do we want to list the corresponding CVEs ?

Reply:
That's a good idea, here's a example of that could look like: https://github.com/grafana/bugbounty/pull/9/commits/1a12991ebbe8ec646291777db9a6675b3f2e9bee